extension/chrome/settings/modules/contacts.ts

```diff
@@ -265,7 +265,7 @@ View.run(class ContactsView extends View {
           }
         }
         container.css('display', 'block');
-        $('#bulk_import .input_pubkey, #bulk_import .action_process, #file_import #fineuploader_button').css('display', 'none');
+        $('#bulk_import .input_pubkey, #bulk_import .action_process, #file_import, #fineuploader_button').css('display', 'none');
```

Comment:
This is the fix for the unrelated issue when the unwanted block was shown after doing bulk import:

![CleanShot 2021-11-13 at 00 34 53@2x](https://user-images.githubusercontent.com/6059356/141585789-d755c1f5-e7e7-4a47-867d-c9587c44ed93.png)